packages/bpk-theming/src/BpkThemeProvider.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ThemeProvider } from 'theming';
+
+import createWithTheme from './createWithTheme';
+import withCssVariables from './withCssVariables';
+
+const BpkThemeProvider = withCssVariables(ThemeProvider);
+
+const disableThemingForComponent = () => {
+  console.log('shaun is cool');
+};
+
+export default BpkThemeProvider;
+const withTheme = createWithTheme();
```

Comment:
Because we are using `var(--bpk-variable-name, $bpk-fallback-color)` in component scss then we don't need to wrap them in `withTheme` HOC's, right?

Reply:
Same as my comment above - it depends if we're going to access `props.theme` inside the components themselves, for validation and stuff. If not, then we can remove all this stuff.

Reply:
Ah ok I get you - yeah we wouldn't wanna add the `theming` dependency to consumers bundles just to validate for props. Its seeming more likely that we'll should do this in the `ThemeProvider` itself.